client/my-sites/email/email-forwarding/email-forwarding-gsuite-details.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card/compact';
+import { domainManagementEmail } from 'my-sites/domains/paths';
+
+const EmailForwardingGSuiteDetails = ( { selectedDomainName, siteSlug, translate } ) => {
+	return (
+		<Card>
+			<p className="email-forwarding__explanation">
+				{ translate(
+					'You are using G Suite with this domain. {{gsuiteLink}}Manage it here{{/gsuiteLink}}.',
```

Comment:
This is most probably just personal preference, but I usually use `{{a}}` instead of named anchors for strings with a single link so translators don't have to deal with this extra noise:

```javascript
You are using G Suite with this domain. {{a}}Manage it here{{/a}}.
```

Reply:
I agree that looks better, and would be more consistent with other places I use components in a translated string as well.